plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -109,26 +112,32 @@ export default function transformProps(chartProps: ChartProps): EchartsTimeserie
             : 0,
       });
   });
+  let [yAxisMin, yAxisMax] = yAxisBounds || [];
+  if (yAxisMin === undefined && contributionMode === 'row' && stack) yAxisMin = 0;
+  if (yAxisMax === undefined && contributionMode === 'row' && stack) yAxisMax = 1;
```

Comment:
This section is fairly opaque and hard to read, wondering if/how much `ts` can help prevent issues around `undef` with more types hints here. Nothing outrageous though.

Reply:
True, will refactor.